inc/Engine/Optimization/AssetsLocalCache.php

```diff
@@ -109,4 +113,55 @@ protected function write_file( $content, $file ) {
 
 		return rocket_put_content( $file, $content );
 	}
+
+	/**
+	 * Check if this link HTML has integrity attribute or not?
+	 *
+	 * @since 3.7.5
+	 *
+	 * @param string $asset Link HTML to be tested.
+	 *
+	 * @return array|false Matched array with integrityhashmethod, integrityhash keys.
+	 */
+	private function has_integrity( $asset ) {
+		if ( ! preg_match( '#\s*integrity\s*=[\'"](?<integrityhashmethod>.*)-(?<integrityhash>.*)[\'"]#Ui', $asset, $integrity_matches ) ) {
+			return false;
+		}
+
+		if ( ! isset( $integrity_matches['integrityhashmethod'] ) || ! isset( $integrity_matches['integrityhash'] ) ) {
```

Comment:
```suggestion
		if ( ! isset( $integrity_matches['integrityhashmethod'], $integrity_matches['integrityhash'] ) ) {
```

You can group multiple checks in one `isset` call, it makes it easier to read